reconcile/utils/semver_helper.py

```diff
@@ -7,7 +7,16 @@ def make_semver(major: int, minor: int, patch: int) -> str:
     return str(semver.VersionInfo(major=major, minor=minor, patch=patch))
 
 
-def parse_semver(version: str) -> semver.VersionInfo:
+def parse_semver(
+    version: str, optional_minor_and_patch: bool = False
+) -> semver.VersionInfo:
+    if optional_minor_and_patch:
+        # semver3 supports optional minor and patch.
+        # until we upgrade to semver3, we support this by adding a default minor and/or patch
```

Comment:
Any blocker to bump version? https://python-semver.readthedocs.io/en/latest/migration/migratetosemver3.html looks ok.

Reply:
```
$ pip install -U semver
unleashclient 5.7.1 requires semver<3.0.0, but you have semver 3.0.2 which is incompatible.
sretoolbox 2.4.2 requires semver~=2.13, but you have semver 3.0.2 which is incompatible.
```
Let's do the semver upgrade later :)